site/src/components/BuildsTable/BuildDateRow.tsx

```diff
@@ -0,0 +1,45 @@
+import { makeStyles } from "@material-ui/core/styles"
+import TableCell from "@material-ui/core/TableCell"
+import TableRow from "@material-ui/core/TableRow"
+import formatRelative from "date-fns/formatRelative"
+
+export interface BuildDateRow {
+  date: Date
+}
+
+export const BuildDateRow: React.FC<BuildDateRow> = ({ date }) => {
+  const styles = useStyles()
+  // We only want the message related to the date since the time is displayed
+  // inside of the build row
+  const displayDate = formatRelative(date, new Date()).split("at")[0]
+
+  return (
+    <TableRow className={styles.buildDateRow}>
+      <TableCell
+        className={styles.buildDateCell}
+        title={date.toLocaleDateString()}
+      >
+        {displayDate}
+      </TableCell>
+    </TableRow>
```

Comment:
This is such a nice, tabbable table! I would love to be able to hit 'enter' and navigate to the log page without having to touch my mouse.